docs/source/ultimate.rst

```diff
@@ -59,6 +59,70 @@ Nous garderons la partie privée de vos données (nom, coordonnées, etc.) dans
+
+Mantendremos la parte privada de sus datos (nombre, información para contactarle, etc.) en un lugar seguro durante al menos <x> años. De esta manera, si uno de los investigadores que utilice los datos de este estudio encuentra algo en sus escaneos cerebrales que pudieran tener un valor diagnóstico, podremos contactar con usted. Después de este periodo de tiempo, destruiremos su información para proteger su privacidad.
+
+**Dejarnos usar y compartir sus datos es voluntario. Sin embargo, tiene que estar dispuesto a compartir sus datos de esta forma para poder participar en este estudio.**
+
+Si no está dispuesto, no podrá participar en el estudio. Al firmar abajo, está aceptando proporcionar sus datos a futuras investigaciones. De esta forma acepta que los datos podrán ser compartidos con otros investigadores de otras instituciones por todo el mundo. Los detalles, resultados, e implicaciones de estos estudios son desconocidos.
+
+Italian
+~~~~~~
+(translation courtesy of Valentina Borghesani)
+
+I dati ed i campioni raccolti in questo studio potrebbero essere utilizzati per altri futuri progetti di ricerca oltre allo studio a cui stai attualmente partecipando. Tali ricerche potranno riguardare qualsiasi argomento, non necessariamente collegato a quello dello studio cui stai partecipando. I dati raccolti, incluse le neuroimmagini, saranno condivisi attraverso una piattaforma online ed una banca dati aperta al grande pubblico.
+
+I dati che verranno condivisi con il pubblico non saranno identificati dal tuo nome ma solo grazie ad un codice numerico, in modo che nessuno possa identificarti o sapere quali dati sono tuoi. Inoltre, al meglio delle nostre conoscenze, non condivideremo altre informazioni che potrebbero essere usate da qualcuno che ti conosce per risalire a quali sono i tuoi dati.
+
+Se tu dovessi cambiare idea e volessi ritirare il tuo consenso alla partecipazione a questo studio (per farlo, chiama <Nome Responsabile> al numero <Numero Telefono>), non raccoglieremo dati aggiuntivi su di te. Se tu dovessi ritirare il tuo consenso prima che i dati siano depositati nella banca dati, questi saranno eliminati. **Tuttavia, dati e risultati eventualmente già condivisi con altri investigatori o con il grande pubblico non possono essere distrutti, rimossi o ritirati.**
```

Comment:
Qualora cambiassi idea, e decidessi di rimuovere il tuo consenso alla partecipazione in questo studio, puoi contattare \<Nome Reponsabile\> al  numero \<Numero Telefono\>. In tal caso non raccoglieremo ulteriori dati su di te.